src/app/api/media/sync/route.ts

```
@@ -0,0 +1,374 @@
+              mediaData.url = value;
+              break;
+            case 'By':
+              mediaData.by = Array.isArray(value) ? value : [];
+              break;
+            case 'Topic':
+              mediaData.topic = Array.isArray(value) ? value : [];
+              break;
+            case 'Thumbnail':
+              mediaData.thumbnail = Array.isArray(value) ? value : [];
+              // Upload thumbnail to Supabase Storage
+              const thumbnailUrl = getThumbnailUrl(value);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)